docs/.vuepress/components/DocVersions.vue

```diff
@@ -15,6 +15,7 @@
 const docsBase= "https://docs.rundeck.com"
 //include the last version in each minor release
 const previousDocsVersions= [
+  "5.0.2",
   "5.0.1", 	
   "5.0.0",
```

Comment:
fyi, you can just change `5.0.0` to `5.0.2` and it will fill in the intermediate ones, you don't have to enumerate the patch versions